### PR TITLE
Update isort version

### PR DESCRIPTION
**Pull request by yhoiseth**
https://github.com/pre-commit/mirrors-isort/releases

### CODE REVIEW

**Comment by asottile** [comment]
I don't plan to keep this number in sync as it invites PRs like this one which are noisy and don't add too much value -- maybe instead just add a comment `#  pick the isort version you'd like to use`

**Comment by yhoiseth** [comment]
> I don't plan to keep this number in sync as it invites PRs like this one which are noisy and don't add too much value -- maybe instead just add a comment `# pick the isort version you'd like to use`

Fixed :)

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ This works especially well when integrated with [`pre-commit`][pre-commit].
     hooks:
     -   id: seed-isort-config
 -   repo: https://github.com/pre-commit/mirrors-isort
-    rev: v4.3.21
```
Comment (asottile) [inline README.md]: heh I didn't mean to delete the current version, just to add the comment

Comment (yhoiseth) [reply README.md]: Fixed :)

**Pull request merged**